rtc/encryption.md

```diff
@@ -0,0 +1,70 @@
+
+* Store a global list of keys for your application, and use the document ID along with random data to salt the key _on your server_ to produce a key unique to the document session. Do not send the salt data to any clients; use `keyHint` to pass around an identifier that clients can use to request the unique key from the server.
+* Use a fixed random key for each document, and generate random salt data to provide a unique key for each session. Pass the salt data to other clients through `keyHint`. The salt data will be stored as plain text by the RTC server, but the salt alone cannot be used to decrypt document data.
+* Generate and store a fixed random key for each document in your database. Ignore both input and response `keyHint` fields.
+
+## Encryption key rotation and key hints
+
+The RTC configuration API is designed to support key rotation. Keys cannot be rotated on demand; if this is important to you, please contact {{site.companyname}} to discuss how we can best provide that functionality.
+
+Document collaboration may be performed in multiple sessions. For example, when a new version of {{site.productname}} is deployed it may be incompatible with existing sessions. Only one session will be active at a time but older sessions may still be used to bootstrap new sessions. As such, old keys cannot be immediately discarded when a new key is requested.
+
+In order to allow for key rotation a key hint is supplied so the encryption provider may tell the difference between these two cases and act accordingly. If the key hint is `null`, then the RTC protocol needs a new key and this is the opportunity for key rotation. If the key hint is not null the protocol is requesting a previously issued key so that it can read the session history.
+
+If the key hint is not included in an encryption provider response for a new key, an empty string will be recorded as the key hint.
+
+> **Warning**: The key hint is stored by the server in plain text. Do not add secret or sensitive information to the key hint.
+
+The key hint can be a key thumbprint, ID, or other non-sensitive identifier that will help select the key, such as a timestamp. It is only recorded when the protocol requests a new key. Returning a key hint from requests for previously issued keys will have no effect.
+
+## RTC encryption details
+
+> **Note**: This section contains the technical details of encryption used to securely transmit document contents. It is provided for information purposes only; understanding these details is not required to integrate RTC.
+
+The RTC protocol does not use the [provided encryption key]({{site.baseurl}}/rtc/configuration#rtc_encryption_provider) to encrypt content directly. Using industry standard cryptography algorithms a unique session key for content encryption is derived from the provided key in a way that makes brute force decryption very difficult.
```

Comment:
```suggestion
The RTC plugin does not use the [provided encryption key]({{site.baseurl}}/rtc/configuration#rtc_encryption_provider) to encrypt content directly. The plugin uses industry standard cryptography algorithms to generate a unique session key for content encryption based on the provided key to improve protection against brute force decryption.
```

Reply:
I appreciate that you're trying to make things easier to follow, but for the target audience here the word "derived" is _very_ important to include. I'll put it back after the suggestions are committed.